Makefile-man.am

```diff
@@ -2600,16 +2605,6 @@ man/systemd-user-sessions.html: man/systemd-user-sessions.service.html
 
 endif
 
-if HAVE_PYTHON
-MANPAGES += \
-	man/systemd.directives.7 \
-	man/systemd.index.7
-MANPAGES_ALIAS += \
-	#
-
```

Comment:
This change unfortunately broke the generation of systemd.directives and systemd.index and needs to be reverted